src/modules/endpoint/DefaultDataContainer.js

```diff
@@ -152,6 +152,7 @@ class DefaultDataContainer implements DataContainer {
   }
 
   getFilePathsFromLanguageResourceCache (languageResourceCache: LanguageResourceCacheStateType): Array<string> {
+    // $FlowFixMe https://github.com/facebook/flow/issues/2221
     return flatMap(
       Object.values(languageResourceCache),
       (file: PageResourceCacheStateType): Array<string> => map(file, ({ filePath }) => filePath)
```

Comment:
Maybe you can extract the `Object.values` part here, so that Flow can at least check for flatMap to have the right type:
```suggestion
    // $FlowFixMe https://github.com/facebook/flow/issues/2221
    const pageResourceCaches: Array<PageResourceCacheStateType> = Object.values(languageResourceCache)
    return flatMap(
      pageResourceCaches
      (file: PageResourceCacheStateType): Array<string> => map(file, ({ filePath }) => filePath)
```
(similar below)